server/httpout/service.go

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2020-present Mattermost, Inc. All Rights Reserved.
+// See License for license information.
+
+package httpout
+
+import (
+	"io"
+
+	"github.com/mattermost/mattermost-server/v5/services/httpservice"
+
+	"github.com/mattermost/mattermost-plugin-apps/server/config"
+)
+
+type Service interface {
+	config.Configurable
+	httpservice.HTTPService
+
+	GetFromURL(url string, trusted bool) ([]byte, error)
+}
+
+type service struct {
+	httpservice.HTTPService
+
+	conf config.Service
+}
+
+var _ config.Configurable = (*service)(nil)
+var _ httpservice.HTTPService = (*service)(nil)
+
+func NewService(conf config.Service) Service {
+	return &service{
+		HTTPService: httpservice.MakeHTTPService(conf.GetMattermostConfig()),
+		conf:        conf,
+	}
+}
+
+func (s *service) Configure(x config.Config) {
+	s.HTTPService = httpservice.MakeHTTPService(s.conf.GetMattermostConfig())
```

Comment:
If a `Service` gets it's configuration form `s.conf` anyway, why does it need to be a `config.Configurable`? Should it create it's `HTTPService` on the fly?

Reply:
That would certainly simplify things, but I figured I should follow the server pattern in case there's caching ever done in the implementation.

Reply:
Good point on caching.

1/5 that it's preferred to blank out variables that are not used.
```suggestion
func (s *service) Configure(_ config.Config) {
	s.HTTPService = httpservice.MakeHTTPService(s.conf.GetMattermostConfig())
```